bindings/matrix-sdk-ffi/src/client_builder.rs

```diff
@@ -260,7 +260,8 @@ pub struct ClientBuilder {
     proxy: Option<String>,
     disable_ssl_verification: bool,
     disable_automatic_token_refresh: bool,
-    cross_process_refresh_lock_id: Option<String>,
+    cross_process_store_locks_holder_name: Option<String>,
+    enable_oidc_refresh_crypto_lock: bool,
```

Comment:
I don't think the refresh lock is anything to do with crypto. It's purely a lock on making a token refresh request.

Reply:
Alrighty. Renamed to `enable_iodc_refresh_lock`.